pkg/operator/api/userconfig/apis.go

```diff
@@ -48,17 +49,36 @@ var apiValidation = &cr.StructValidation{
 			},
 		},
 		{
-			StructField: "ModelPath",
-			StringPtrValidation: &cr.StringPtrValidation{
-				Validator: cr.GetS3PathValidator(),
-			},
+			StructField:      "ExternalModel",
+			StructValidation: externalModelFieldValidation,
 		},
 		apiComputeFieldValidation,
 		tagsFieldValidation,
 		typeFieldValidation,
 	},
 }
 
+type ExternalModel struct {
+	Path   string `json:"path" yaml:"path"`
+	Region string `json:"region" yaml:"region"`
+}
+
+var externalModelFieldValidation = &cr.StructValidation{
+	DefaultNil: true,
+	StructFieldValidations: []*cr.StructFieldValidation{
+		{
+			StructField: "Path",
+			StringValidation: &cr.StringValidation{
+				Validator: cr.GetS3PathValidator(),
+			},
+		},
+		{
+			StructField:      "Region",
+			StringValidation: &cr.StringValidation{},
```

Comment:
Should this be `Required: true` or have a default? Vishal went with default of us-west-2 in his PR: https://github.com/cortexlabs/cortex/commit/9a51405d761489b7989ec9817efabf27136bc065#diff-69b6039f0ce969b3871ddda857887583R160